documentation/blog/2024-10-01-whats-new-in-svelte-october-2024.md

```diff
@@ -0,0 +1,95 @@
+---
+title: "What's new in Svelte: October 2024"
+description: 'Better each blocks and a callstack for infinite loops'
+author: Dani Sandoval
+authorURL: https://dreamindani.com
+---
+
+Thanks to the number of folks who have been trying the Svelte 5 release candidates, the maintainers have been hard at work in addressing bugs and implementing performance improvements for the upcoming release. This month, we'll cover the most notable changes along with the many apps, resources and libraries that the community has created over the last few weeks.
+
+## What's new in Svelte
+
```

Comment:
the migration tool is now available. `npx svelte-migrate svelte-5` to test it out

Reply:
one I was excited about is that the output got a lot smaller for people who are using `@sveltejs/enhanced-img`: https://github.com/sveltejs/svelte/pull/13075